src/utils.js

```diff
@@ -27,6 +27,21 @@ class Utils {
     return newName;
   }
 
+  resolvePath(...values) {
+    for (let i = 0; i < values.length; i++) {
+      let value = values[i];
+
+      if (value) {
+        if (value.path) {
+          value = value.path;
+        }
+
+        return value;
+      }
+    }
+    return false;
```

Comment:
As I see it, the function should either return a valid value, or `false` if it fails to do so. encodeURIComponent will never return false. Since the input to the function is `let redirectTo = utils.resolvePath(this.props.redirectTo, router.getAuthenticatedHomeRoute(), router.getHomeRoute(), '/');`. I.e. there's a default '/' value that is the last argument.